apps/OboeTester/app/src/main/java/com/mobileer/oboetester/StreamConfigurationView.java

```diff
@@ -469,12 +469,24 @@ void updateDisplay(StreamConfiguration actualConfiguration) {
         mActualChannelMaskView.setText(StreamConfiguration.convertChannelMaskToText(value));
 
         boolean isMMap = actualConfiguration.isMMap();
-        mStreamInfoView.setText("burst = " + actualConfiguration.getFramesPerBurst()
-                + ", capacity = " + actualConfiguration.getBufferCapacityInFrames()
-                + ", devID = " + actualConfiguration.getDeviceId()
-                + ", " + (actualConfiguration.isMMap() ? "MMAP" : "Legacy")
-                + (isMMap ? ", " + StreamConfiguration.convertSharingModeToText(sharingMode) : "")
-        );
+
+        String msg = "";
+        msg += "burst = " + actualConfiguration.getFramesPerBurst();
+        msg += ", capacity = " + actualConfiguration.getBufferCapacityInFrames();
+        msg += ", devID = " + actualConfiguration.getDeviceId();
+        msg += ", " + (actualConfiguration.isMMap() ? "MMAP" : "Legacy");
+        msg += (isMMap ? ", " + StreamConfiguration.convertSharingModeToText(sharingMode) : "");
+
+        int hardwareChannelCount = actualConfiguration.getHardwareChannelCount();
+        int hardwareSampleRate = actualConfiguration.getHardwareSampleRate();
+        int hardwareFormat = actualConfiguration.getHardwareFormat();
+        msg += "\nhardware channel count = " + (hardwareChannelCount ==
```

Comment:
Screen real estate is very tight in this view.
Please try to fit on one line, eg:
"Hardware: #ch=2, format=3, SR=96000"
or
"HW: #ch=?, format=?, SR=?"